content/home/download.md

```diff
@@ -68,19 +68,20 @@ subtitle = ""
 {{% col "col-12 col-sm-4" %}}
 <div class="featurette-icon"><i class="fab fa-linux"></i></div>
 **GMT源码**:
-[v6.1.1](http://mirrors.ustc.edu.cn/gmt/gmt-6.1.1-src.tar.gz)
+[v6.2.0](http://mirrors.ustc.edu.cn/gmt/gmt-6.2.0-src.tar.gz)
 <br>
 **GSHHG数据**:
 [v2.3.7](http://mirrors.ustc.edu.cn/gmt/gshhg-gmt-2.3.7.tar.gz)
 <br>
 **DCW数据**:
-[v1.1.4](http://mirrors.ustc.edu.cn/gmt/dcw-gmt-1.1.4.tar.gz)
+[v2.0.0](http://mirrors.ustc.edu.cn/gmt/dcw-gmt-2.0.0.tar.gz)
```

Comment:
```suggestion
[v2.0.0](https://github.com/GenericMappingTools/dcw-gmt/releases/download/2.0.0/dcw-gmt-2.0.0.tar.gz)
```

Reply:
Why does ustc mirror not sync dcw2.0.0？

Reply:
How about gshhg?

Reply:
The idea is to fully remove the FTP server. Future GSHHG versions also won't be uploaded to the FTP.

Reply:
Shall we change it too?

Reply:
GSHHG v2.3.7 is still available on the USTC mirror. It's definitely faster than github.